src/json_ui.cc

```diff
@@ -19,7 +19,7 @@ namespace Kakoune
 {
 
 struct invalid_rpc_request : runtime_error {
-    invalid_rpc_request(String message)
+    invalid_rpc_request(const String& message)
```

Comment:
I think the idiom that is used in the codebase is to pass a `StringView` rather than a constant reference?

Reply:
I think StringView is about generalizing interfaces to work for `const char*`, `String` and various proxy types.  This only has one caller that passes a String, so using a const reference avoids a copy.  Granted, this copy is super unimportant, since it's for an exception path, and I'm not attached to it.  StringView would construct a temporary object here.